### PR TITLE
Fix badges order on profile sidebar

### LINKED ISSUE #3724: Blank space in badge section on Profile
**Describe the bug**

Badge not placed properly in the account page. Looks like the badge image width cause the issue. The whitespace is show in some account only.

**To Reproduce**

 
 1. Go to https://dev.to/ben
 2. Check the badges widget

**Expected behavior**

**Desktop (please complete the following information):**

- OS: MacOs
- Browser: Chrome 
- Version: 76.0.3809.100
 
**Additional context**

The space issue is on the mobile version too.

### PR DESCRIPTION
**Pull request by nobitagit**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [X] Bug Fix
- [ ] Documentation Update

## Description
Make sure the layout is consistent when rendering badges on the sidebar

## Related Tickets & Documents

fixes #3724 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Screenshot 2019-08-16 at 00 32 20

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/views/users/_sidebar_additional.html.erb [hunk 1]
```diff
@@ -22,8 +22,6 @@
         </div>
       </a>
     <% end %>
-    <% if @user.badge_achievements_count > 0 %>
-      <%= render "articles/badges_area" %>
-    <% end %>
+    <%= render "articles/badges_area" %>
```
Comment (nobitagit) [inline app/views/users/_sidebar_additional.html.erb]: ^ This check was actually duplicated just inside the `badges_area` template ([here](https://github.com/thepracticaldev/dev.to/pull/3729/files#diff-ae027a3a438869c94fbb7f5ce2fcf576R1)), so I picked one and remove the other

**Comment by nobitagit** [comment]
Uhm, I thought the randomness given by the rotating the images was enough, but I see what you mean.

In that case what about leaving the grid as is but applying a random vertical spacing to the images? This would avoid gaps such as the one that lead to opening the issue in the first place but would perhaps feel closer to what you mean. For example:

image

Otherwise open to any suggestion or to close this PR, no problem of course!

**Comment by jessleenyc** [comment]
Digging it!

**Comment by benhalpern** [comment]
@nobitagit @jessleenyc The original thing wasn't really considered a bug or expected behavior, it's just kind of how things were. I'm fairly neutral on the subject, but I could see how the past behavior could bother some folks. All in all, I feel pretty good about merging it, so will do!

**Pull request merged**